.travis.yml

```diff
@@ -94,6 +94,13 @@ script:
       node scripts/facts-tracker/index.js \
         "fiber-tests" "$FIBER_TESTS"
 
+      # Should consolidate this with the above fiber run
+      echo 'Testing in fiber mode against test records...'
+      scripts/fiber/record-tests
+      git --no-pager diff scripts/fiber
+      FIBER_TESTS_STATUS=$(git status --porcelain scripts/fiber)
+      test -z "$FIBER_TESTS_STATUS"
```

Comment:
It's good for the team but might be noisy for third-party PR. Not too worried about it though. Let's see how it works out.